src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -750,11 +750,17 @@ void TableDockWidget::exportGroupsToSpreadsheet() {
     
     if ( samples.size() == 0) return;
 
+    if( sFilterSel == groupsSCSV || sFilterSel == groupsSTAB || sFilterSel == groupsCSV || sFilterSel == groupsTAB )
+     _mainwindow->getAnalytics()->hitEvent("Export Groups to CSV", "Export Groups");
```

Comment:
@46navdeep Can you change the name of the category to just "Exports" i.e "Export Groups to CSV" to "Exports"

Reply:
@sahil21 Done.